ci-operator/config/kata-containers/runtime/kata-containers-runtime-master.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  base:
+    name: "4.5"
+    namespace: ocp
+    tag: base
+binary_build_commands: .ci/openshift-ci/build.sh
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- context_dir: .ci/openshift-ci/images
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/kata-containers/runtime/_out
+  to: kata-installer
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.5"
+  namespace: ocp
+tests:
+- artifact_dir: /go/src/github.com/kata-containers/tests/_output/tests/openshift-e2e
+  as: e2e-periodic-azure-nested-virt
+  commands: |
+    KATA_WORKDIR=/go/src/github.com/kata-containers/tests
+    cd $KATA_WORKDIR
+    .ci/openshift-ci/test.sh
+  cron: 2 7 * * *
+  openshift_installer_custom_test_image:
```

Comment:
Maybe you want to consider moving to [multistage build](https://steps.ci.openshift.org/help/) in favor of using the old installer template approach. IIUC the install template approach is regarded as deprecated. I could create a [PoC on our existing nightly job](https://github.com/openshift/release/pull/11764) to prove that this also works (not sure about your case though).

Reply:
Adding this CI for Kata Containers took me more time than expected so at this point I only would like to see it running regardless of this deprecated approach. Then I can work on improvements/updates later on.

Anyway, thanks for pointing out this new multistage build approach.

Reply:
Well, we are going to deprecate the templates soon, so at this point, we don't allow users to use templates for adding new jobs.

Reply:
Ok, so I am going to label this to wip and start the migration work. Thanks!